python/pyiceberg/catalog/hive.py

```diff
@@ -197,17 +234,41 @@ def __init__(self, name: str, properties: Properties, uri: str):
         super().__init__(name, properties)
         self._client = _HiveClient(uri)
 
-    def _convert_hive_into_iceberg(self, table: HiveTable) -> Table:
-        # Requires reading the manifest, will implement this in another PR
-        # Check the table type
-        return Table(identifier=(table.dbName, table.tableName))
+    def _convert_hive_into_iceberg(self, table: HiveTable, metadata_location: Optional[str] = None) -> Table:
+        properties: Dict[str, str] = table.parameters
+        if TABLE_TYPE not in properties or properties[TABLE_TYPE] != ICEBERG:
```

Comment:
Can you make the table type check case insensitive? We've had problems with that in the past.

Reply:
I've made the value case-insensitive. Let me know if you also want to do the key.

Reply:
No need to alter the key. Just the value is fine.